test_matrix.py

```diff
@@ -0,0 +1,9 @@
+from matrix import Matrix
+
+def test_add_method(add_result: Matrix, matrix_one: Matrix, matrix_two: Matrix):
+    """Function for testing __add__ method."""
+    assert (matrix_one + matrix_two).get_matrix() == add_result.get_matrix()
```

Comment:
I think you can overload `__eq__` method to _equal_ two matrices.
With something like this:
```python
    def __eq__(self: Self, other: Self) -> bool:
        """Equality method."""
        return self._matrix == other._matrix
```